.github/workflows/release.yaml

```diff
@@ -8,30 +8,43 @@ jobs:
   release:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout code
+      - name: Checkout Code
```

Comment:
lets keep word as sentence where only first letter is capital

Reply:
I don't mind. My reasoning is that this is a name or a title which generally has every word capitalized

Reply:
done